src/Plant/constants.jl

```diff
@@ -1,12 +1,77 @@
-using CLIMAParameters
-
 # set the default floating type, Float32 for now. Should be the same as in the RT module
-FT = Float64
-
-# more pending, may need to migrate this to CLIMAParameters in the future
-ρ_H₂O = FT(998.0) # Kg m⁻³ | @ 293.15 K, may need to calculate it from temperature in the future
+const FT = Float32
 
 # load the constant gravity value
 struct EarthParameterSet <: AbstractEarthParameterSet end
-earth   = EarthParameterSet()
-const gravity = CLIMAParameters.Planet.grav(earth)
+const earth   = EarthParameterSet()
+const gravity = FT( CLIMAParameters.Planet.grav(earth) )
+const GAS_R   = FT( gas_constant() )
+
+# pre-defined ΔE and ΔP to avoid repeated FT conversion
+const ΔEP_2 = FT(1e-2)
+const ΔEP_3 = FT(1e-3)
+const ΔEP_4 = FT(1e-4)
+const ΔEP_5 = FT(1e-5)
+const ΔEP_6 = FT(1e-6)
+const ΔEP_7 = FT(1e-7)
+
+# pre-defined number to avoid repeated FT conversion in the code
+const NUMB_0   = FT( 0.0)
```

Comment:
Using `FT` in the code is fine, and it's not always needed. Please read the [conversion and promotion](https://docs.julialang.org/en/v1/manual/conversion-and-promotion/#conversion-and-promotion-1) docs to get a better idea of when they're needed. Defining module-global constants like this should be avoided.